frontend/src/modules/logs/EventType.tsx

```diff
@@ -63,27 +57,25 @@ export const getEventDetails = (event: LogEvent, eventType: EventType): string =
     }
     return '';
   } else if (event.type === LogEventType.system) {
-    if (event.code === LogEventCode.mcu_backend_connection_down) {
-      return 'Hardware controller lost connection';
-    }
-    if (event.code === LogEventCode.backend_mcu_connection_down) {
-      return 'Backend server lost connection from hardware controller';
-    }
-    if (event.code === LogEventCode.backend_frontend_connection_down) {
-      return 'User interface lost connection';
-    }
-    if (event.code === LogEventCode.frontend_backend_connection_down) {
-      return 'User interface lost connection';
-    }
-    if (event.code === LogEventCode.mcu_backend_connection_up) {
-      return 'Hardware controller connected';
-    }
-    if (event.code === LogEventCode.backend_frontend_connection_up) {
-      return 'User interface connected';
-    }
-    if (event.code === LogEventCode.charger_disconnected) {
-      return 'Battery charger is disconnected';
+    const eventDetailsMap = new Map<LogEventCode, string>([
```

Comment:
If we can move our `Map`s out to file-level constants, let's do that.

Reply:
added constants.ts in src/modules/logs

Reply:
I really like the approach of moving them into a separate file, I hadn't thought of that! It makes sense here because the `Map`s are so big.